xml/ceph_maintenance_updates.xml

```diff
@@ -33,6 +33,145 @@
+       be warned again if additional repairs are performed, you can provide a
+       value to the command and specify the value of
+       <option>mon_osd_warn_num_repaired</option>. This command will be replaced
+       in future releases by the health mute/unmute feature.
+     </para>
+   </listitem>
+   <listitem>
+     <para>
+       It is now possible to specify the initial MON to contact for &ceph;
+       tools and daemons using the <option>mon_host_override config</option> option
+       or <option>--mon-host-override <replaceable>IP</replaceable></option>
+       command-line switch. This generally should only be used for debugging
+       and only affects initial communication with &ceph;’s MON cluster.
+     </para>
+   </listitem>
+   <listitem>
+     <para>
+       Fix an issue with osdmaps not being trimmed in a healthy cluster.
+     </para>
+   </listitem>
+ </itemizedlist>
+ <bridgehead renderas="sect1">&cephname; 14.2.11 Point Release</bridgehead>
+ <para>
+   In addition to bug fixes, this major upstream release brought a number of
+   notable changes:
+ </para>
+ <itemizedlist>
+   <listitem>
+     <para>
+       RGW: The <command>radosgw-admin</command> sub-commands dealing with
+       orphans – <command>radosgw-admin orphans find</command>, <command>radosgw-admin orphans finish</command>,
+       <command>radosgw-admin orphans list-jobs</command> – have been deprecated.
+       They have not been actively maintained and they store intermediate
+       results on the cluster, which could fill a nearly-full cluster. They have
+       been replaced by a tool, currently considered experimental,
+       <command>rgw-orphan-list</command>.
+     </para>
+   </listitem>
+   <listitem>
+     <para>
+       Now, when <option>noscrub</option> and/or <option>nodeep-scrub</option> flags
+       are set globally or per pool, scheduled scrubs of the type disabled will
+       be aborted. All user initiated scrubs are <emphasis>not</emphasis> interrupted.
+     </para>
+   </listitem>
+   <listitem>
+     <para>
+       Fixed a ceph-osd crash in committed OSD maps when there is a failure
+       to encode the first incremental map.
+     </para>
+   </listitem>
+ </itemizedlist>
+ <bridgehead renderas="sect1">&cephname; 14.2.10 Point Release</bridgehead>
+ <para>
+  This upstream release patched one security flaw:
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    CVE-2020-10753: rgw: sanitize newlines in s3 CORSConfiguration’s ExposeHeader
+   </para>
+  </listitem>
+</itemizedlist>
+ <para>
+   In addition to security flaws, this major upstream release brought a number of
+   notable changes:
+ </para>
+ <itemizedlist>
+   <listitem>
+     <para>
+       RGW: Bucket notifications now support Kafka endpoints. This requires
+       librdkafka of version 0.9.2 and up. Note that Ubuntu 16.04.6 LTS
+       (Xenial Xerus) has an older version of librdkafka, and would require
+       an update to the library.
```

Comment:
This "Kafka endpoints" note is not applicable to our product, so it should be dropped.